bugbug/repository.py

```diff
@@ -81,6 +86,12 @@ def _transform(commit):
     # Covert to a list, as a set is not JSON-serializable.
     obj['types'] = list(obj['types'])
 
+    components_touched = []
+    for fl in patch_data.keys():
+        if COMPONENTS.get(fl):
+            components_touched.extend(COMPONENTS[fl])
```

Comment:
The data is in this format:
```
  "Cargo.lock": [
    "Firefox Build System", 
    "General"
  ],
```

The list is not a list of components. The first element of the list is the product (`Firefox Build System` in this case), and the second element of the list is the component (`General` in this case).

We want the full component, so it should be the concatenation of product and component with `::` in the middle (in this case `Firefox Build System::General`).